namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-domain-events-dev/resources/hmpps-interventions.tf

```diff
@@ -0,0 +1,134 @@
+resource "kubernetes_secret" "intervention_global_events_sns" {
+  metadata {
+    name      = "hmpps-domain-events-topic"
+    namespace = var.namespace
+    # Remove when namespace has been migrated
+    # namespace = "hmpps-interventions-dev"
```

Comment:
🙋‍♂️ This has been migrated already. Would you prefer me to open a PR separately?

Reply:
ah hadn't spotted this, guess it is because the live-1 namespace hasn't been removed yet.  I'll fix

Reply:
thanks!